docker-compose.yml

```diff
@@ -1,8 +1,7 @@
 version: "3"
 services:
   mysql-1:
-    image: percona:5.7
-    platform: linux/x86_64
+    image: docker.io/mysql/mysql-server:8.0
```

Comment:
Do we not want to test on 5.7 locally anymore? It seems valuable to have that option..?

Reply:
We are testing 5.7 in the github workflows as a secondary CI pipeline.